api-docs-slate/source/includes/_node_rpc.md

```diff
@@ -1,53 +1,63 @@
 # RPC Calls - Node
 
 ```shell--curl
-curl $url/ \
-  -H 'Content-Type: application/json' \
+curl $url \
   -X POST \
-  --data '{ "method": "methodname", "params": [...] "id": "some-id" }'
+  --data '{ "method": "<method>", "params": [...] "id": "some-id" }'
 ```
 
 ```shell--cli
-bcoin-cli rpc params...
+bcoin-cli rpc <method> <params>
 ```
 
 ```javascript
 const {NodeClient} = require('bclient');
-const rpc = new NodeClient({
-  network: 'testnet'
-});
+const {Network} = require('bcoin');
+const network = Network.get('regtest');
 
-(async () => {
-  const res = await rpc.execute('MethodName', [ ...params ]);
+const clientOptions = {
+  network: network.type,
+  port: network.rpcPort,
+  apiKey: 'api-key'
+}
 
-  // RES will return "result" part of the object, not the id or error
-  // error will be thrown.
-})().catch((err) => {
-  console.error(err.stack);
-});
-```
+const client = new NodeClient(clientOptions);
 
-> The above command returns JSON structured like this:
+(async () => {
+  const result = await client.execute('<method>', [ <params> ]);
+  console.log(result);
+})();
+```
 
-> Further examples will only include "result" part.
+> The above cURL command returns JSON structured like this:
 
 ```json
 {"result": resultObject ,"error": errorObject, "id": passedID}
 ```
 
+> Further examples will only include "result" part.
+> CLI and Javascript calls return just the "result" or an error.
+
+
+
 
-Bcoin rpc calls mimic Bitcoin Core's RPC.
-This is documentation how to use it with `bcoin`.
+bcoin RPC calls mimic Bitcoin Core's RPC.
 
 RPC Calls are accepted at:
 `POST /`
 
-*Note: bcoin-cli rpc and javascript will return error OR result.*
+*Notes:*
+
+*bcoin-cli rpc and Javascript will either return an error OR the result.*
+
+*Javascript result will return the "result" part of the object, not the id or error*
+
+*If a Javascript error is encountered it will be thrown instead of returned in JSON*
```

Comment:
This is a good addition. Might be worth mentioning somewhere that if you have enabled logging on your node, you can check for errors there (this is something we often have to ask for when people report problems on github, slack or irc, i.e. They say "something didn't work. What do I do?" and then we have to ask if the logs say anything)